components/compass-runtime-agent/internal/kyma/util.go

```diff
@@ -10,22 +10,24 @@ import (
 func createAssetFromEventAPIDefinition(eventAPIDefinition model.EventAPIDefinition) clusterassetgroup.Asset {
 
 	return clusterassetgroup.Asset{
-		ID:      eventAPIDefinition.ID,
-		Name:    eventAPIDefinition.Name,
-		Type:    getEventApiType(eventAPIDefinition.EventAPISpec),
-		Content: getEventSpec(eventAPIDefinition.EventAPISpec),
-		Format:  clusterassetgroup.SpecFormat(getEventSpecFormat(eventAPIDefinition.EventAPISpec)),
+		ID:          eventAPIDefinition.ID,
+		Name:        eventAPIDefinition.Name,
+		DisplayName: eventAPIDefinition.Description,
+		Type:        getEventApiType(eventAPIDefinition.EventAPISpec),
+		Content:     getEventSpec(eventAPIDefinition.EventAPISpec),
+		Format:      clusterassetgroup.SpecFormat(getEventSpecFormat(eventAPIDefinition.EventAPISpec)),
 	}
 }
 
 func createAssetFromAPIDefinition(apiDefinition model.APIDefinition) clusterassetgroup.Asset {
 
 	return clusterassetgroup.Asset{
-		ID:      apiDefinition.ID,
-		Name:    apiDefinition.Name,
-		Type:    getApiType(apiDefinition.APISpec),
-		Content: getSpec(apiDefinition.APISpec),
-		Format:  getSpecFormat(apiDefinition.APISpec),
+		ID:          apiDefinition.ID,
+		Name:        apiDefinition.Name,
+		DisplayName: apiDefinition.Description,
```

Comment:
Can you use name from APIDefinition/EventAPIDefinition? 
<img width="226" alt="Screen Shot 2020-04-07 at 10 23 13" src="https://user-images.githubusercontent.com/20790348/78647866-711a7c00-78bb-11ea-82c3-a518edd92994.png">
<img width="980" alt="Screen Shot 2020-04-07 at 10 22 00" src="https://user-images.githubusercontent.com/20790348/78647871-724ba900-78bb-11ea-85f0-7887c3bc7c84.png">

Reply:
Done.